src/api/app.js

```diff
@@ -69,10 +69,7 @@ module.exports.authorise = (appToken, permissions, options) => {
 module.exports.connectAuthorised = (appToken, authUri) => {
   return getObj(appToken)
     .then((app) => app.auth.loginFromURI(authUri))
-    .then((connectedApp) => {
-      freeObj(appToken);
-      return genHandle(connectedApp);
-    });
+    .then((connectedApp) => appToken);
```

Comment:
safe_app_nodejs doesn't return the same instance while calling `loginFromURI` API. [here](https://github.com/maidsafe/safe_app_nodejs/blob/master/src/app.js#L173) It creates a new app instance and returns that back.